src/ert/gui/main.py

```diff
@@ -40,8 +40,14 @@ def run_gui(args: Namespace, plugin_manager: ErtPluginManager | None = None) ->
     signal(SIGINT, SIG_DFL)
 
     QDir.addSearchPath("img", str(files("ert.gui").joinpath("resources/gui/img")))
-
+    qcolor_space_category = QLoggingCategory(
+        "QColorSpace"
+    )  # This category is to remove noise from the QColorSpace warnings on QApplication creation
+    qcolor_space_category.setFilterRules("*.warning=false")
     app = QApplication(["ert"])  # Early so that QT is initialized before other imports
+    qcolor_space_category.setFilterRules(
+        ""
+    )  # No longer needed after creation of QApplication
```

Comment:
This comment might be misleading. Here we reset the filter to enable warnings again.